Sources/TuistGenerator/Generator/ProjectGenerator.swift

```diff
@@ -263,25 +263,51 @@ final class ProjectGenerator: ProjectGenerating {
                        pbxproj: PBXProj,
                        project: Project,
                        graph: Graphing) throws -> GeneratedProject {
-        var generatedProject: GeneratedProject!
-
-        try FileHandler.shared.inTemporaryDirectory { temporaryPath in
-
+        let fileHandler = FileHandler.shared
```

Comment:
I don't think we need to define this here. We can access `FileHandler.shared` directly.

Reply:
The intention was to reduce some of the line lengths below where it was used